LondonBritishLibrary/orient/BLorient810.xml

```diff
@@ -52,48 +121,71 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">192</measure>
-                                    <dimensions type="outer" unit="mm">
-                                        <height>375</height>
-                                        <width>305</width>
+                                    <measure unit="leaf" type="blank">2</measure>
+                                    <dimensions type="outer" unit="in">
+                                        <height>14.75</height>
+                                        <width>12</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+                                <foliation>One unnumbered blank leaf at the beginning
+                                and one at the end.</foliation>
+
                             </supportDesc>
-                            
+
                             <layoutDesc>
-                                <layout columns="3" writtenLines="30" cert="high"/>
+                                <layout columns="3" writtenLines="30"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
                                 <desc>The manuscript is well written, between <date>1721</date> and <date>1730</date>
                                 </desc>
-                                
-                                <date notBefore="1721" notAfter="1730" resp="PRS10171Wright"/>
-                                
+                                <date notBefore="1721" notAfter="1730"/>
                             </handNote>
-                            
-                        </handDesc>   
+                        </handDesc>
                         <additions>
-                            
+                          <list>
+                            <item xml:id="a1">
+                              <locus target="#2r"></locus>
+                              <desc type="GuestText"><title xml:lang="gez">
+                              ጸዋትወ፡ ሕማማት፡ ዘእግዚእነ፡ ኢየሱስ፡ ክርስቶስ።
+                              <foreign xml:lang="en">The Degrees of the Sufferings
+                              of Our Lord</foreign></title></desc>
+                            </item>
+                            <item xml:id="a2">
+                              <locus target="#2v"></locus>
+                              <desc type="OwnershipNote">The note states that the
+                              book <foreign xml:lang="gez">አቡሻክር፡</foreign> belonged to
```

Comment:
I would mark the title up as the title, if we later want to filter for books where the title is noted - or at least keep the wording of Wright here (a note containing the name of the book)